graders/python/install.sh

```diff
@@ -8,7 +8,10 @@ yum install -y \
     make \
     dos2unix \
     graphviz \
-    graphviz-devel
+    graphviz-devel \
+    rubygems
+
+gem install mustache
```

Comment:
Ah, much better! OK. I've updated this and the `python_autograder/mustache_process.sh` file to use chevron. I tested this both on my computer and in a docker container, and it seems to be working correctly.

I personally may find some incompatibilities in my already-created problems btw chevron and mustache, but it should be no big deal for others adopting this fresh!

Note that I used the current-since-Jan-2021 0.14.0 version of chevron in `requirements.txt`. I'm guessing that's old enough to be a reasonable choice.